tests/Microsoft.Identity.Web.Test/Resource/ScopesRequiredAuthorizeAttributeTests.cs

```diff
@@ -14,8 +14,8 @@
 
 namespace Microsoft.Identity.Web.Test.Resource
 {
-    [RequiredScope(RequiredScopesConfigurationKey="AzureAd:Scopes")]
-    [RequiredScope(ScopeOne)]
+    [RequiredScopeAttribute(RequiredScopesConfigurationKey="AzureAd:Scopes")]
```

Comment:
Attribute is not needed here. Just   `[RequiredScope(RequiredScopesConfigurationKey="AzureAd:Scopes")]`